5calls/app/src/main/java/org/a5calls/android/a5calls/controller/StatsActivity.java

```diff
@@ -90,9 +91,13 @@ private void initializeUI(DatabaseHelper db) {
         callCountHeader.setText(getStringForCount(
                 mCallCount, R.string.your_call_count_one, R.string.your_call_count));
 
-        List<Long> contacts = db.getCallTimestampsForType(RepCallActivity.CONTACTED);
-        List<Long> voicemails = db.getCallTimestampsForType(RepCallActivity.VOICEMAIL);
-        List<Long> unavailables = db.getCallTimestampsForType(RepCallActivity.UNAVAILABLE);
+        List<Long> contacts = db.getCallTimestampsForType(OutcomeView.CONTACTED);
```

Comment:
I think we need to discuss this further with @nickoneill; I've started on Slack. Stats shouldn't have all this special-case work, instead we should have an enum as part of the outcomes JSON that maps each custom outcome to our existing categories. Then we won't log what button the user clicked, instead we will keep logging the type of outcome (vm/skip/contact). Otherwise logs will get diluted with things like "supports" or "opposes" or "agrees", etc, which isn't useful for a user's stats.

Reply:
Done. See:

https://github.com/5calls/android/pull/89/files?diff=split#diff-a3bc9f38b50e18c62b12e978cb7ad48fR81